### DISCUSSION TITLE
my project run success on dev mode but when when i want run output i have this error

### DISCUSSION THREAD

**Discussion by ezioadf2** [nuxt/nuxt#16969] [Questions] [answered]
`[nuxt] [request error] Cannot find module 'C:\nostrud\.output\server\node_modules\@hapi\hoek\lib\assert' imported from C:\nostrud\.output\server\chunks\server.mjs
Did you mean to import @hapi/hoek/lib/assert.js?
  at new NodeError (node:internal/errors:372:5)
  at finalizeResolution (node:internal/modules/esm/resolve:437:11)
  at moduleResolve (node:internal/modules/esm/resolve:1009:10)
  at defaultResolve (node:internal/modules/esm/resolve:1218:11)
  at ESMLoader.resolve (node:internal/modules/esm/loader:580:30)
  at ESMLoader.getModuleJob (node:internal/modules/esm/loader:294:18)
  at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:80:40)
  at link (node:internal/modules/esm/module_job:78:36)`

hi when i build my project and i run index.mjs i have top error
what i can do?

package.json:

`{
    "private": true,
    "scripts": {
        "dev": "nuxi dev",
        "build": "nuxi build",
        "start": "node .output/server/index.mjs"
    },
    "devDependencies": {
        "nuxt": "npm:nuxt3@latest",
        "nuxt-windicss": "latest"
    },
    "dependencies": {
        "chart.js": "^3.7.1",
        "crypto-hash": "^2.0.1",
        "decimal.js": "^3.0.0",
        "jalali-moment": "^3.3.11",
        "joi": "^17.6.0",
        "js-cookie": "^3.0.1",
        "mitt": "^3.0.0",
        "moment-jalaali": "^0.9.4",
        "path": "^0.12.7",
        "swiper": "^8.3.2",
        "vue-chart-3": "^3.1.2",
        "vue-jalali-moment": "^1.0.0",
        "vue-recaptcha-v3": "^2.0.1",
        "vue3-google-oauth2": "^1.0.6",
        "vue3-persian-datetime-picker": "^1.2.2"
    }
}
`

**Comment by danielroe** [comment, marked as answer]
You can track in https://github.com/nuxt/framework/issues/6652. For now you can likely work around by adding to your `build.transpile.`
